src/components/FormScrollView.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {ForwardedRef} from 'react';
+import {ScrollView} from 'react-native';
+import styles from '../styles/styles';
+
+type FormScrollViewProps = Partial<ScrollView> & {
```

Comment:
This looks wrong, it makes `FormScrollViewProps` to have such fields as `componentWillMount` available. Shouldn't it be `ScrollViewProps` ? Also, why `Partial` if it actually gets passed to ScrollView?
```suggestion
type FormScrollViewProps = ScrollViewProps & {
```
```

Reply:
Good catch! Adjusted